### DISCUSSION TITLE
为什么被回复 Issue 的时候如果重复，他们能精确加入准确的issue编号？

### DISCUSSION THREAD

**Discussion by Ad-closeNN** [Meloong-Git/PCL#2996] [提问] [answered]
有些 issue比这个issue要前好多，是因为记忆好还是另有其法？

**Comment by WForst-Breeze** [comment, marked as answer]
大部分靠搜索，有些常见的重复的多了就记住了
这里大部分 collaborator 都是在这个仓库刚开没太久就开始做这个工作了，所以见的 issue 比较多🤔

**Comment by MoYuan-CN** [comment]
咱们有文档（ #1930 [#1930镜像](https://pcl-doc.rth1.app) ）
确实看多了就记住了

**Reply by iPodtouch6557** [reply]
手机端适配有没有做好？

**Reply by MoYuan-CN** [reply]
没。。
去我那开个 Issue 罢。。我找个时间给他做了去。。

**Reply by WorldHim** [reply]
> 手机端适配有没有做好？

手机端还是 GitHub Mobile 用着舒服

**Reply by MoYuan-CN** [reply]
等着，今天给他搓了去
